rapid7_insightvm_cloud/icon_rapid7_insightvm_cloud/actions/start_scan/action.py

```diff
@@ -25,7 +25,7 @@ def run(self, params={}):
         if ips or hostnames:
             body = _format_body(hostnames, ips)
             resources = self.connection.ivm_cloud_api.call_api("assets", "POST", params, body)
-            extra_ids = resources[1].get("data")
+            extra_ids = resources.get("data")
             for extra_id in extra_ids:
```

Comment:
Will `resources` always have a `data` key? If not, then line 29 will fail due to attempted iteration over `None`. A safer way to write line 28 would be `extra_ids = resources.get("data", [])`, that way line 29 will not crash and will just get skipped over due to no elements in the list.